<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
   "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
	<head>
	   	<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
		<meta http-equiv="Content-Language" content="English" />
		<meta name="Author" content="Nuno D. Mendes" />
		<meta name="Description" content="Homepage of Nuno D. Mendes" />
		<link rel="stylesheet" type="text/css" href="stylesheet.css" />
		<title>Nuno D. Mendes</title>
	</head>

	<body>
		<div class="name"><a href="index.html">Nuno D. Mendes</a></div>
		<table class="body"><tr><td align="left">
			<table class="header">
				<tr><td class="header">
					<a href="index.html">Index</a> - 
					<a href="publications.html">Publications</a> -
							Software -
							<a href="documents.html">Documents</a>
				</td></tr>
			</table>

			<!-- *********************************************************************** -->
			<hr/>
			<!-- *********************************************************************** -->			
			<h2>Software</h2>

			<fieldset>
				<legend>Firefront/Avatar</legend>
				<b>URL:</b> <a href="http://compbio.igc.gulbenkian.pt/nmd/node/59">http://compbio.igc.gulbenkian.pt/nmd/node/59</a></br>
				FIREFRONT/AVATAR are two methods for the identification of the set of attractors reachable from a given initial condition, and the evaluation of their probability, over asynchronous discrete dynamics. FIREFRONT is a quasi-exact method, simultaneously following all (concurrent) trajectories while propagating state probabilities. AVATAR is a modified Monte Carlo algorithm in order to handle strongly connected components. It avoids revisiting the same states, due to transient and terminal cycles, by rewiring the state transition graph.
			</fieldset>

			<fieldset>
				<legend>CRAVELA</legend>
				<b>URL:</b> <a href="http://cravela.org">http://cravela.org</a> (currently inaccessible)</br>
				The CRAVELA framework provides a means to predict pre-miRNA candidates in single genomes, with an optional analysis of its conservation in phylogenetically close species.	
				 
				It may also incorporate multiple RNAseq data to predict mature sequences and estimate their expression across different experimental conditions. The predicted pre-miRNA candidates are thus also evaluated by their expression score.
				The framework incorporates the annotation information, when available, to eliminate unlikely candidates, and it provides stem-loop structural analysis methods for metazoan genomes.
			</fieldset>


			<fieldset>
				<legend>GinSIM -- Composition</legend>
				<b>URL:</b> <a href="http://ginsim.org/beta">http://ginsim.org/beta</a></br>
				GinSIM is a tool for the analysis and specification of logical regulatory networks.
			</fieldset>

			<fieldset>
				<legend>Yeastract-Discoverer</legend>
				<b>URL:</b><a href="http://yeastract.com/discoverer/">http://yeastract.com/discoverer/</a></br>
				DISCOVERER provides a set of tools that can be used to identify complex motifs found to be over-represented in the promoter regions of co-regulated genes. DISCOVERER is based on the MUSA and RISO algorithms. These algorithms take as input a list of genes and identify over-represented motifs, which can then be compared with transcription factor binding sites described in the YEASTRACT database.
			</fieldset>

			<fieldset>
				<legend>MUSA</legend>
				<b>URL:</b><a href="http://kdbio.inesc-id.pt/~ndm/software/musa.html">http://kdbio.inesc-id.pt/~ndm/software/musa.html</a></br>
				MUSA is a parameter-free, heuristic method for the extraction of conserved patterns in a set of DNA input sequences.
			</fieldset>
            
            <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
            <script src="common.js"></script>
		</body>
</html>
