<html><head><title>Practical Class</title><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">ol{margin:0;padding:0}.c9{max-width:468pt;background-color:#ffffff;padding:72pt 72pt 72pt 72pt}.c0{text-indent:36pt;height:11pt;direction:ltr}.c4{height:11pt;direction:ltr}.c6{color:inherit;text-decoration:inherit}.c3{direction:ltr;margin-left:36pt}.c1{color:#1155cc;text-decoration:underline}.c7{text-indent:36pt}.c10{font-style:italic}.c5{direction:ltr}.c8{margin-left:36pt}.c2{font-family:"Courier New"}.title{widows:2;padding-top:0pt;line-height:1.15;orphans:2;text-align:left;color:#000000;font-size:21pt;font-family:"Trebuchet MS";padding-bottom:0pt;page-break-after:avoid}.subtitle{widows:2;padding-top:0pt;line-height:1.15;orphans:2;text-align:left;color:#666666;font-style:italic;font-size:13pt;font-family:"Trebuchet MS";padding-bottom:10pt;page-break-after:avoid}li{color:#000000;font-size:11pt;font-family:"Arial"}p{color:#000000;font-size:11pt;margin:0;font-family:"Arial"}h1{widows:2;padding-top:10pt;line-height:1.15;orphans:2;text-align:left;color:#000000;font-size:16pt;font-family:"Trebuchet MS";padding-bottom:0pt;page-break-after:avoid}h2{widows:2;padding-top:10pt;line-height:1.15;orphans:2;text-align:left;color:#000000;font-size:13pt;font-family:"Trebuchet MS";font-weight:bold;padding-bottom:0pt;page-break-after:avoid}h3{widows:2;padding-top:8pt;line-height:1.15;orphans:2;text-align:left;color:#666666;font-size:12pt;font-family:"Trebuchet MS";font-weight:bold;padding-bottom:0pt;page-break-after:avoid}h4{widows:2;padding-top:8pt;line-height:1.15;orphans:2;text-align:left;color:#666666;font-size:11pt;text-decoration:underline;font-family:"Trebuchet MS";padding-bottom:0pt;page-break-after:avoid}h5{widows:2;padding-top:8pt;line-height:1.15;orphans:2;text-align:left;color:#666666;font-size:11pt;font-family:"Trebuchet MS";padding-bottom:0pt;page-break-after:avoid}h6{widows:2;padding-top:8pt;line-height:1.15;orphans:2;text-align:left;color:#666666;font-style:italic;font-size:11pt;font-family:"Trebuchet MS";padding-bottom:0pt;page-break-after:avoid}</style></head><body class="c9"><p class="c5"><span>1. Apply the Needleman-Wunsch algorithm to the following sequences</span></p><p class="c5"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p><p class="c3"><span>1: </span><span class="c2">ATCGATC</span></p><p class="c3"><span>2: </span><span class="c2">GATCATT</span></p><p class="c4"><span></span></p><p class="c5"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(a) Calculate the dynamic programming matrix.</span></p><p class="c4"><span></span></p><p class="c3"><span>(b) What is the score of the alignment?</span></p><p class="c4 c8"><span></span></p><p class="c3"><span>(c) What is the optimal alignment?</span></p><p class="c4 c8"><span></span></p><p class="c3"><span>(c) How would you modify the Needleman-Wunsch algorithm to detect the best overlap between two sequences (semi-global alignment). </span><span class="c10">Note that this amounts to ignoring gaps at the extremities of the alignment.</span></p><p class="c4"><span></span></p><p class="c5"><span>2. Download the the files </span><span class="c1"><a class="c6" href="http://kdbio.inesc-id.pt/~ndm/molbiosys/some-pre-let-7.fa">some-pre-let-7.fa</a></span><span>&nbsp;and</span><span class="c1"><a class="c6" href="http://kdbio.inesc-id.pt/~ndm/molbiosys/tRNAs.fa">&nbsp;tRNAs.fa</a></span><span>&nbsp;</span></p><p class="c4"><span></span></p><p class="c7 c5"><span>(a) Use </span><span class="c1"><a class="c6" href="http://www.ebi.ac.uk/Tools/msa/clustalo/">ClustalOmega</a></span><span>&nbsp;to align the let-7 sequences using the RNA option</span></p><p class="c0"><span></span></p><p class="c7 c5"><span>(b) Use </span><span class="c1"><a class="c6" href="http://rna.informatik.uni-freiburg.de/LocARNA/Input.jsp">LocARNA</a></span><span>&nbsp;to fold and align the let-7 sequences.</span></p><p class="c0"><span></span></p><p class="c7 c5"><span>(c) Is the structural alignment useful in this case?</span></p><p class="c0"><span></span></p><p class="c5 c7"><span>(d) Repeat the previous steps using the tRNA sequences.</span></p><p class="c0"><span></span></p><p class="c7 c5"><span>(e) Is the structural alignment useful in this case?</span></p><p class="c4"><span></span></p><p class="c5"><span>3. Download the </span><span class="c1"><a class="c6" href="http://kdbio.inesc-id.pt/~ndm/molbiosys/upstream.fa">fasta</a></span><span>&nbsp;file containing the upstream region of several bacterial genes known to be regulated by the AbgR TF</span></p><p class="c4"><span></span></p><p class="c5"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(a) Use </span><span class="c1"><a class="c6" href="http://www.ebi.ac.uk/Tools/msa/clustalo/">ClustalOmega</a></span><span>&nbsp;to align the sequences</span></p><p class="c5"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p><p class="c5"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(b) Is it useful to identify common motifs?</span></p><p class="c4"><span></span></p><p class="c5"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(c) Use the MUSA motif finder located in your virtual machine.</span></p><p class="c4"><span></span></p><p class="c5"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(d) Use </span><span class="c1"><a class="c6" href="http://meme.nbcr.net/meme/cgi-bin/meme.cgi">MEME</a></span><span>&nbsp;selecting &lsquo;any number of repetitions&rsquo; and &lsquo;search given strand only&rsquo;.</span></p><p class="c4"><span></span></p><p class="c5"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(e) Compare the results.</span></p><p class="c4"><span></span></p><p class="c4"><span></span></p></body></html>